src/backend/distributed/operations/worker_shard_copy.c

```diff
@@ -88,6 +107,8 @@ CanUseLocalCopy(uint32_t destinationNodeId)
 }
 
 
+#define REPLICATION_ORIGIN_CMD_BUFFER_SIZE 1024
```

Comment:
we usually put constants at the top

Reply:
Removed this now, using StringInfo() instead